"Enhancement compilation must be configured in AVA’s Babel options" error with TypeScript.

I'm trying to run some unit tests with TypeScript using `ts-node` package as described in official ava's TS [recipe](https://github.com/avajs/ava/blob/master/docs/recipes/typescript.md).

Facing an error:
```
> node ./node_modules/ava/cli.js

  ✖ Enhancement compilation must be configured in AVA’s Babel options.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! some_package@0.0.1 test: `node ./node_modules/ava/cli.js`
npm ERR! Exit status 1
```

Node version: 13.6.0
Ava version: 3.0.0
TypeScript version: 3.7.5
OS: Linux Mint 19.1 (Tessa)
Running ava both ways leads to same result: 
`node ./node_modules/ava/cli.js` as locally installed
`ava` as globally installed.

Must note that with ava v2.4.0 everything works as expected.
Repository to reproduce: https://github.com/meyve/ava_test

--- a/docs/recipes/typescript.md
+++ b/docs/recipes/typescript.md
@@ -38,7 +38,6 @@ Once installed, add the `tsconfig-paths/register` entry to the `require` section
 ```json
 {
 	"ava": {
-		"compileEnhancements": false,
 		"extensions": [
 			"ts"
 		],

--- a/docs/recipes/typescript.md
+++ b/docs/recipes/typescript.md
@@ -10,31 +10,11 @@ This guide assumes you've already set up TypeScript for your project. Note that 
 
 You can configure AVA to recognize TypeScript files. Then, with `ts-node` installed, you can compile them on the fly.
 
-There are some minor changes between AVA version 3 and previous versions of AVA. In V3, the `compileEnhancements` configuration is no longer necessary unless you are planning to use AVA's Babel support, as AVA no longer ships with it's Babel setup.
-
-### AVA Version 3
-
 `package.json`:
 
 ```json
 {
 	"ava": {
-		"extensions": [
-			"ts"
-		],
-		"require": [
-			"ts-node/register"
-		]
-	}
-}
-```
-
-### Other AVA Versions
-
-```json
-{
-	"ava": {
-		"compileEnhancements": false,
 		"extensions": [
 			"ts"
 		],

--- a/docs/recipes/typescript.md
+++ b/docs/recipes/typescript.md
@@ -10,7 +10,26 @@ This guide assumes you've already set up TypeScript for your project. Note that 
 
 You can configure AVA to recognize TypeScript files. Then, with `ts-node` installed, you can compile them on the fly.
 
+There are some minor changes between AVA version 3 and previous versions of AVA. In V3, the `compileEnhancements` configuration is no longer necessary unless you are planning to use AVA's Babel support, as AVA no longer ships with it's Babel setup.
+
+### AVA Version 3
+
 `package.json`:
+
+```json
+{
+	"ava": {
+		"extensions": [
+			"ts"
+		],
+		"require": [
+			"ts-node/register"
+		]
+	}
+}
+```
+
+### Other AVA Versions
 
 ```json
 {

